libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -9,15 +9,37 @@
         {{ b.authors.join(',') }}
       </div>
     </div>
-    <div>
-      <button
+    <div *ngIf="b.finished" data-testing="finished-date" tabindex="0"
+         [attr.aria-label]="'Finished on' + b.finishedDate" class="reading-list-item--details--author">
+            Finished on: {{ b.finishedDate }}
```

Comment:
The date here is rendered as the ISO string. Maybe we could use an Angular functionality to transform it into a more readable format?

Reply:
Date format is corrected